### PR TITLE
V technically only needs `git`

### PR DESCRIPTION
**Pull request by AxiomApollo**
Yes you need a C compiler, but if you don't V will handle that for you, and you only need other C compilers for optimized binary output.

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -20,15 +20,17 @@ The major way to get the latest and greatest V, is to __install it from source__
 It is __easy__, and it usually takes __only a few seconds__.
 
 ### Linux, macOS, FreeBSD, etc:
-You need `git`, and a C compiler like `tcc`, `gcc` or `clang`, and `make`:
+You need [`git`](https://git-scm.com/downloads)
+>You can install and use a C compiler of your choice for binary optimization, but by default V will use [TCC](http://savannah.nongnu.org/projects/tinycc/).
```
Comment (dumblob) [inline doc/docs.md:24]: `... by default V will use ...` -> `... by default V will download and use ...` (so that any deployment is clear)

Comment (AxiomApollo) [reply doc/docs.md:24]: ahh yes true true

File: doc/docs.md [hunk 2]
```diff
@@ -20,15 +20,21 @@ The major way to get the latest and greatest V, is to __install it from source__
 It is __easy__, and it usually takes __only a few seconds__.
 
 ### Linux, macOS, FreeBSD, etc:
-You need `git`, and a C compiler like `tcc`, `gcc` or `clang`, and `make`:
+You need [`git`](https://git-scm.com/downloads)
```
Comment (spytheman) [inline doc/docs.md:23]: Do not remove references for the C compiler; on some platforms (apple m1 or freebsd) tcc does not work, or we do not supply prebuild binaries.

Comment (AxiomApollo) [reply doc/docs.md:23]: ahh i see, well that fixes that then.

Comment (spytheman) [reply doc/docs.md:23]: Can you restore the previous sentence?

Comment (AxiomApollo) [reply doc/docs.md:23]: I think I'll just close this since it seems somehow I deleted the fork I made.

**Pull request closed without merging**